adapters/unicredit-adapter/src/main/java/de/adorsys/xs2a/adapter/service/impl/UnicreditAccountInformationService.java

```diff
@@ -0,0 +1,89 @@
+package de.adorsys.xs2a.adapter.service.impl;
+
+import de.adorsys.xs2a.adapter.adapter.BaseAccountInformationService;
+import de.adorsys.xs2a.adapter.http.StringUri;
+import de.adorsys.xs2a.adapter.service.GeneralResponse;
+import de.adorsys.xs2a.adapter.service.RequestHeaders;
+import de.adorsys.xs2a.adapter.service.StartScaProcessResponse;
+import de.adorsys.xs2a.adapter.service.ais.ConsentCreationResponse;
+import de.adorsys.xs2a.adapter.service.ais.ConsentStatusResponse;
+import de.adorsys.xs2a.adapter.service.ais.Consents;
+import de.adorsys.xs2a.adapter.service.impl.mapper.ScaStatusResponseMapper;
+import de.adorsys.xs2a.adapter.service.impl.mapper.UnicreditCreateConsentResponseMapper;
+import de.adorsys.xs2a.adapter.service.impl.mapper.UnicreditStartAuthorisationResponseMapper;
+import de.adorsys.xs2a.adapter.service.impl.model.UnicreditAccountScaStatusResponse;
+import de.adorsys.xs2a.adapter.service.impl.model.UnicreditStartScaProcessResponse;
+import de.adorsys.xs2a.adapter.service.model.ScaStatusResponse;
+import de.adorsys.xs2a.adapter.service.model.TransactionAuthorisation;
+import de.adorsys.xs2a.adapter.service.model.UpdatePsuAuthentication;
+
+import java.util.Map;
+
+public class UnicreditAccountInformationService extends BaseAccountInformationService {
+    private static final String AUTHENTICATION_CURRENT_NUMBER_QUERY_PARAM = "authenticationCurrentNumber";
+    private static final String PSU_IP_ADDRESS = "PSU-IP-Address";
+    private static final String MOCK_PSU_IP_ADDRESS = "0.0.0.0";    // TODO should be changed to a real data
```

Comment:
Perhaps it should be DEFAULT instead of MOCK and used only when no value is provided by the client.

Reply:
Fixed, thanks!